src/database-postgresql.cpp

```diff
@@ -0,0 +1,505 @@
+/*
+Copyright (C) 2014 jjb, James Dornan <james at catch22 dot com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "config.h"
+
+#include "database-postgresql.h"
+#include "exceptions.h"
+#include "settings.h"
+#include "log.h"
+#include "debug.h"
+#include "filesys.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+Database_PostgreSQL::Database_PostgreSQL(const std::string &savedir)
+{
+
+	m_savedir = savedir;
+
+	bool succeeded = m_conf.readConfigFile(std::string((m_savedir) + DIR_DELIM + "world.mt").c_str());
```

Comment:
This feature was added recently, this code is likely (loosely) based on the old version of the redis backend.